### ISSUE TITLE
怎么在使用bmtrain训练的时候读取已经训练好的增量微调的权重？？

### ISSUE THREAD

**Issue by liweiqing1997** [OpenBMB/BMTrain#89]
我尝试了多种方式：
1 model.load_state_dict(torch.load(args.model_path), strict=False)
2 bmt.load(model, args.LoRA_path,strict=False)
但是，打印模型参数后发现并没有被读取进去。为什么发生这样的情况。

# load model
bmt.init_distributed(seed=0)
config = CPMAntConfig.from_json_file(args.config_path)
model = CPMAntPlus(config=config)
bmt.load(model, args.model_path,strict=True)

# insert LoRA
#delta_model = AutoDeltaModel.from_finetuned(args.LoRA_path, backbone_model=model)
delta_model = LoraModel(backbone_model=model, modified_modules=["project_q", "project_v"], backend="bmt")
delta_model.freeze_module(exclude=["deltas"], set_state_dict=True)
